src/game/utils.js

```diff
@@ -38,4 +38,13 @@ export const areAllConnected = points => {
     points = points.filter(isContained(current));
   }
   return points.length === 0;
-}
\ No newline at end of file
+}
+
+export const calculateVector = (source, destination) => {
+  return ({x: destination.x-source.x,
+    y: destination.y-source.y,
+    z: destination.z-source.z});
+}
+
+export const getPointByVector =(point, vector) =>
```

Comment:
Rename to addVector perhaps?

Reply:
The name isn't perfect, but the idea is to calculate a new point in the coordinate system based on existing point and vector. `addVector` suggests that there is some matrix operatoin(ish)? We should ask some physics guys what they think :atom:

Reply:
Hahaha. How about we just rename the first argument v1, and the second v2, and imagine that all vectors that we call it with can be described as originating in origo?

Reply:
I tell you what, I will resolve this once the indentation is fixed, ok? ;)